templates/reminder-no-maintainer.mail

```diff
@@ -0,0 +1,28 @@
+Dear port maintainers,
+
+The portscout new distfile checker has detected that one or more unmaintained 
+ports appear to be out of date. Please take the opportunity to check each of the 
+ports listed below, and if possible and appropriate, submit/commit an update. 
+Please consider also adopting this port. If any ports have already been updated, 
+you can safely ignore the entry.
```

Comment:
This bit is a tad confusing. What are we trying to say and in what circumstances does it apply? Do we mean, `when you go and check, and port has already been updated, just ignore this?`

If so, wording may be better earlier in the paragraph as, replacing:

`Please take the opportunity to check each of the ports listed below, and if possible and appropriate`

with

`Please take the opportunity to check each listed port, and where still out-of-date, ....`

Reply:
I do agree that it is maybe a bit confusing. However, my only contribution to this paragraph is `Please consider also adopting this port.` It's worth discussing if the wording could be changed but it doesn't really have anything to do with this PR.

Reply:
If this is just the existing single template text pulled out into a separate template that's fine 👍